engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/activation-key-details.controller.js

```diff
@@ -51,9 +51,13 @@ angular.module('Bastion.activation-keys').controller('ActivationKeyDetailsContro
             ];
         };
 
+        $scope.setOriginalEnvironment = function(newEnvironment) {
+            $scope.originalEnvironment = newEnvironment;
+        };
+
         $scope.activationKey = ActivationKey.get({id: $scope.$stateParams.activationKeyId}, function (activationKey) {
-            $scope.$broadcast('activationKey.loaded', activationKey);
```

Comment:
What did the `$scope.$broadcast do`?

Reply:
It was responsible for sending an event that another scope could act upon. See above controller => `$scope.$on('activationKey.loaded', function () {` no longer needed!